remote_monitoring_demo/recommendations_infra_demo/recommendations_demo.sh

```diff
@@ -246,6 +246,7 @@ function monitoring_demo_start() {
 	elif [[ ${getMetricsRecommendations} -eq 1 ]]; then
  		echo "Generating the metrics and recommendations for all experiments available in kruize"
  		get_metrics_recommendations
+		get_metrics_boxplots
```

Comment:
Please update the readme with steps to run this and validate the output.

Reply:
Can you also resolve the conflicts

Reply:
Done.